includes/class-amp-theme-support.php

```diff
@@ -2376,6 +2318,69 @@ public static function prepare_response( $response, $args = [] ) {
 		return $response;
 	}
 
+	/**
+	 * Optimizer instance to use.
+	 *
+	 * @param array $args Associative array of arguments to pass into the transformation engine.
+	 * @return Optimizer\TransformationEngine Optimizer transformation engine to use.
+	 */
+	private static function get_optimizer( $args ) {
+		$configuration = self::get_optimizer_configuration( $args );
+
+		// @todo Replace CurlRemoteRequest with a Requests version?
+		$remote_request = new CachedRemoteRequest( new CurlRemoteRequest() );
```

Comment:
Just a thought that since AMP is on a weekly release cycle, perhaps a week would be a better TTL?

```suggestion
		$remote_request = new CachedRemoteRequest( new CurlRemoteRequest(), 24 * 7 );
```

I don't feel strongly about this, however. A daily TTL will minimize lag.

Reply:
I don't feel strongly at all about this either. I can happily adapt to a week if you prefer that (less bandwidth/more lag). Just let me know which one to use.

Reply:
@sebastianbenz What do you recommend? Even though AMP is on a weekly release cycle, hotfix releases could be made in-between, yes? Therefore 1 day is probably better.